DEVELOPMENT.md

```diff
@@ -1,101 +1,39 @@
 # Development
 
-This document is __a work-in-progress__ which aims to describe various tips for __developing and working with this repository itself__.  It is not intended as a guide on how to use the project within another project, which should be covered elsewhere in the project's documentation.
+## Workflow
 
-For now, this will be a collection of tips and tricks for the maintainers of the project.
+> When in doubt, reference the `scripts` in `package.json`.
 
-## Staging a release as a PR
+To start development, simply run `npm install` (`npm i` for short). This will install and build the project.
 
-Depending on the size of the release, it may be ideal to have a staging PR which gathers the various features we're intending on releasing into the version.  This can be a great way for the community to understand the scope of a release and a clear way for maintainers to publicize it and gather feedback.
+To run Typescript in watch mode, run `npm run watch`.
 
-1. Create a branch off `main` named `release-X.Y.Z`, where `X.Y.Z` is the intended release.
-2. Edit the `CHANGELOG.md`, removing the `vNEXT` section entirely.  This section will remain on `main` and be resolved during the merge.
-3. Add a new section for `### vX.Y.Z` with a bullet indicating that something is coming soon:
+If you have [Volta](https://docs.volta.sh/guide/getting-started) installed, your Node and NPM versions will be configured automatically for you.
 
-   ```
-   ### vX.Y.Z
+## Testing
 
-   - Nothing yet! Stay tuned.
-   ```
+To run tests, run `npm test` (or `npm t` for short).
 
-4. Commit this change so GitHub will allow a PR to be opened against `main` with a notable change.  A suggested commit message is `Prepare CHANGELOG.md for upcoming vX.Y.Z release.`
-5. Push the branch to GitHub
-6. On GitHub, open a PR from the new release branch which targets `main`.
-   __For the title of the PR__, use "Release X.Y.Z".  __For the body,__ use the contents of the template in the `.github/APOLLO_RELEASE_TEMPLATE.md` file in this repository.  Adjust the body as you see necessary.
+Arguments can be passed to the test command using the `--` like so: `npm t -- --watch` (this runs Jest in watch mode).
+Similarly, you can run specific test files by name like so: `npm t -- ApolloServer.test.ts`.
 
-## Publishing a release
+Smoke tests run separately from Jest tests. They can be run with `npm run test:smoke`.
```

Comment:
Maybe explain that smoke tests are specifically about ensuring that the built tarballs work properly. (You do mention it later under build system I guess)